components/email_aliases/email_aliases.mojom

```diff
@@ -35,6 +35,13 @@ union GenerateAliasResult {
   string error_message;
 };
 
+struct AliasesUpdatedInfo {
```

Comment:
if you want to handle these in the same message maybe this should be a union? Otherwise might be better to have a separate function